prettyPrinters/lldb/__init__.pyi

```diff
@@ -0,0 +1,379 @@
+from typing import Any, List
+
+import formatters
+
+
+# Some useful interfaces from https://github.com/llvm/llvm-project/tree/master/lldb/bindings/interface
```

Comment:
404 when I try to open the page

Reply:
Thanks for catching that, the branch was renamed [recently](https://foundation.llvm.org/docs/branch-rename/), and now `main` should be used instead

Reply:
To prevent such things, you may use concrete commit instead of branch name